{
  "HepMC3": [],
  "NLopt": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2254979"
  ],
  "PyQt4": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2248128"
  ],
  "RBTools": [],
  "WALinuxAgent": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246085"
  ],
  "Zim": [],
  "afflib": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2260973"
  ],
  "ara": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279553"
  ],
  "argparse-manpage": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279987"
  ],
  "awscli2": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271404"
  ],
  "b43-tools": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245249"
  ],
  "btrfs-progs": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245650"
  ],
  "buildstream": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2252071"
  ],
  "ceph": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259639"
  ],
  "collectd": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247197"
  ],
  "condor": [],
  "csound": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271396"
  ],
  "dionaea": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247008"
  ],
  "docker-compose": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247015"
  ],
  "eralchemy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280551"
  ],
  "eric": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279980"
  ],
  "fail2ban": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2244834"
  ],
  "firmitas": [],
  "flatpak-module-tools": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2252072"
  ],
  "fmf": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279994"
  ],
  "fontforge": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245798"
  ],
  "freeradius": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245785"
  ],
  "future": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250662"
  ],
  "gtts": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246380"
  ],
  "jc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247039"
  ],
  "jrnl": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280580"
  ],
  "jsonnet": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245815"
  ],
  "kernel": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2264420"
  ],
  "kitty": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2265524"
  ],
  "koji": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280552"
  ],
  "krita": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2278646"
  ],
  "lasso": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2256949"
  ],
  "libaccounts-glib": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261300"
  ],
  "libarrow": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250849"
  ],
  "libprelude": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247193"
  ],
  "libpreludedb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261338"
  ],
  "libsbml": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2262374"
  ],
  "libuser": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247195"
  ],
  "linkchecker": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279995"
  ],
  "m2crypto": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245806"
  ],
  "mod_wsgi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245872"
  ],
  "mpsolve": [],
  "muse": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272918"
  ],
  "nut": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2248125"
  ],
  "nwchem": [],
  "nwg-panel": [],
  "opencv": [],
  "openscap": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2260316"
  ],
  "openvswitch": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250661"
  ],
  "paraview": [],
  "petsc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250673"
  ],
  "plplot": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272975"
  ],
  "py-radix": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259528"
  ],
  "pybind11": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2252083"
  ],
  "pycairo": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2264851"
  ],
  "pychromecast": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272929"
  ],
  "pyflakes": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279989"
  ],
  "pygame": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250670"
  ],
  "pyliblo": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2248131"
  ],
  "pyodbc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246290"
  ],
  "pyotherside": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245796"
  ],
  "pyserial": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245859"
  ],
  "pytest": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2275051"
  ],
  "python-Traits": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250844"
  ],
  "python-ZConfig": [],
  "python-agate": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271392"
  ],
  "python-aiosmtpd": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2263017"
  ],
  "python-amico": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2244829"
  ],
  "python-anyio": [],
  "python-anymarkup-core": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261545"
  ],
  "python-astroid": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2254528"
  ],
  "python-astunparse": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279984"
  ],
  "python-asyncssh": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2251916"
  ],
  "python-audioread": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2248130"
  ],
  "python-avocado": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250847"
  ],
  "python-backrefs": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246376"
  ],
  "python-bash-kernel": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280810"
  ],
  "python-beartype": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2265487"
  ],
  "python-boltons": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259631"
  ],
  "python-boto3": [],
  "python-bottle": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245642"
  ],
  "python-bsddb3": [],
  "python-cachelib": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2251780"
  ],
  "python-capturer": [],
  "python-catkin_lint": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259550"
  ],
  "python-cclib": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2276581"
  ],
  "python-configargparse": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2265132"
  ],
  "python-css-parser": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245651"
  ],
  "python-cssutils": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2252065"
  ],
  "python-curio": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246053"
  ],
  "python-devicely": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2277745"
  ],
  "python-devtools": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280965"
  ],
  "python-dill": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2264225"
  ],
  "python-dipy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280596"
  ],
  "python-djvulibre": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226179"
  ],
  "python-domdf-python-tools": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259552"
  ],
  "python-etcd": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246060"
  ],
  "python-exdir": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2266601"
  ],
  "python-extras": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259522"
  ],
  "python-flask-basicauth": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261560"
  ],
  "python-flask-httpauth": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2221959"
  ],
  "python-flask-mailman": [],
  "python-frozenlist": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2275060"
  ],
  "python-ftfy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2278647"
  ],
  "python-futurist": [],
  "python-gast": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279988"
  ],
  "python-gbulb": [],
  "python-gensim": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261825"
  ],
  "python-glue": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2221960"
  ],
  "python-google-cloud-monitoring": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280563"
  ],
  "python-graph-tool": [],
  "python-greenlet": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245874"
  ],
  "python-h5io": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2264873"
  ],
  "python-hl7": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247023"
  ],
  "python-hypercorn": [],
  "python-ibm-vpc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261572"
  ],
  "python-imagecodecs": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226212"
  ],
  "python-imagehash": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226213"
  ],
  "python-immutables": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246142"
  ],
  "python-iniparse": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245852"
  ],
  "python-intbitset": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247036"
  ],
  "python-ipdb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280968"
  ],
  "python-ipyparallel": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280811"
  ],
  "python-jedi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280581"
  ],
  "python-jenkins-job-builder": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247035"
  ],
  "python-jinja2": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245265"
  ],
  "python-jnius": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272966"
  ],
  "python-json-logger": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279986"
  ],
  "python-jsonformatter": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271060"
  ],
  "python-jupyter-kernel-singular": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280961"
  ],
  "python-jupyter-kernel-test": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280962"
  ],
  "python-libcst": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280604"
  ],
  "python-libusb1": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280555"
  ],
  "python-lmdb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259530"
  ],
  "python-logutils": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226233"
  ],
  "python-makefun": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245884"
  ],
  "python-matplotlib-venn": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280562"
  ],
  "python-mechanize": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279550"
  ],
  "python-memray": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280960"
  ],
  "python-mglob": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2244827"
  ],
  "python-mimeparse": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245643"
  ],
  "python-minimock": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2244826"
  ],
  "python-mirakuru": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2275066"
  ],
  "python-multiprocess": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246347"
  ],
  "python-munch": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2275057"
  ],
  "python-numexpr": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250671"
  ],
  "python-oauthlib": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247024"
  ],
  "python-openoffice": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2244824"
  ],
  "python-openpaperwork-core": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245800"
  ],
  "python-orderedset": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226269"
  ],
  "python-pathvalidate": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259547"
  ],
  "python-patsy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280967"
  ],
  "python-pdir2": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2275052"
  ],
  "python-pexpect": [],
  "python-pillow": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280556"
  ],
  "python-pip": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279985"
  ],
  "python-pivy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245811"
  ],
  "python-ppft": [],
  "python-precis_i18n": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246305"
  ],
  "python-psycopg2": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245673"
  ],
  "python-py-gql": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246126"
  ],
  "python-pydantic": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2256753"
  ],
  "python-pydantic-core": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259626"
  ],
  "python-pyfastnoisesimd": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250674"
  ],
  "python-pymilter": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259637"
  ],
  "python-pypet": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2277855"
  ],
  "python-pyside6": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2266591"
  ],
  "python-pytest-qt": [],
  "python-pytest-testinfra": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279552"
  ],
  "python-pytest-trio": [],
  "python-pytn3270": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246275"
  ],
  "python-pyzstd": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2276579"
  ],
  "python-qcustomplot-pyqt": [],
  "python-readability-lxml": [],
  "python-recordclass": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2265475"
  ],
  "python-requests-cache": [],
  "python-rich": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271397"
  ],
  "python-rtmidi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226321"
  ],
  "python-ruffus": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259546"
  ],
  "python-scss": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245678"
  ],
  "python-setuptools-rust": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279993"
  ],
  "python-setuptools_scm_git_archive": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2240107"
  ],
  "python-smartcols": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226335"
  ],
  "python-sphinx": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271405"
  ],
  "python-sphinx-notfound-page": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261607"
  ],
  "python-sphinxcontrib-openapi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279996"
  ],
  "python-stack-data": [],
  "python-stompest": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280565"
  ],
  "python-straight-plugin": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2244822"
  ],
  "python-tblib": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271387"
  ],
  "python-testinfra": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279554"
  ],
  "python-tomlkit": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2264860"
  ],
  "python-toolz": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245886"
  ],
  "python-tpm2-pytss": [],
  "python-trio": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246354"
  ],
  "python-trio-websocket": [],
  "python-trustme": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272940"
  ],
  "python-twisted": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2255342"
  ],
  "python-typecode": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261610"
  ],
  "python-typeguard": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2275067"
  ],
  "python-uritools": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245671"
  ],
  "python-usort": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271063"
  ],
  "python-vedo": [],
  "python-wadllib": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245644"
  ],
  "python-wcmatch": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246283"
  ],
  "python-webob": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245641"
  ],
  "python-webpy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2256932"
  ],
  "python-webscrapbook": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2252057"
  ],
  "python-wsaccel": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2265482"
  ],
  "python-x3dh": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261614"
  ],
  "python-xmlsec": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261617"
  ],
  "python-yappi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2252054"
  ],
  "python-zbase32": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261619"
  ],
  "python-zope-exceptions": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280553"
  ],
  "python-zope-testrunner": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2254185",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2251779"
  ],
  "python3-zope-fixers": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246285"
  ],
  "pythran": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280964"
  ],
  "pyzor": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245827"
  ],
  "qgis": [],
  "qpid-dispatch": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245601"
  ],
  "quodlibet": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259638"
  ],
  "remmina": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247196"
  ],
  "rmlint": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272923"
  ],
  "root": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2278644"
  ],
  "rpkg": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2276896"
  ],
  "rpkg-util": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2276893"
  ],
  "rss2email": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245637"
  ],
  "swift-lang": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245653"
  ],
  "tiled": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261754"
  ],
  "transtats-cli": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2264228"
  ],
  "ubertooth": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259655"
  ],
  "vrpn": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247291"
  ],
  "wesnoth": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280558"
  ]
}